features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -191,11 +191,12 @@ export function SidebarNavigation() {
           </LinkList>
 
           <List>
-            <MenuItemButton
+            <MenuItemLink
               text="Support"
               iconSrc="/icons/support.svg"
               isCollapsed={isSidebarCollapsed}
-              onClick={() => alert("Support")}
+              href="mailto:support@prolog-app.com?subject='Support Request:'"
```

Comment:
Exactly 😄

Reply:
sorry for that! I got lost in the wild lol!✌